src/modules/esl-utils/async/test/debounce.test.ts

```diff
@@ -40,12 +40,12 @@ describe('async/debounce', () => {
   test('cancel debounce', () => {
     const fn = jest.fn();
     const debounced = debounce(fn, 10);
-    const promise = debounced.promise;
```

Comment:
That was the actual reason for an exception.
The promise should be extracted after the first call.